src/main/java/com/google/maps/model/PlacesSearchResult.java

```diff
@@ -0,0 +1,69 @@
+package com.google.maps.model;
+
+import java.net.URL;
+
+/**
+ * <p>PlaceSearchResult represents a single result in the search results return from the Google Places API Web
+ * Service.</p>
+ *
+ * <p>Please see <a href="https://developers.google.com/places/web-service/search#PlaceSearchResults">Place
+ * Search Results</a> for more detail.</p>
+ */
+public class PlacesSearchResult {
```

Comment:
As discussed IRL, yes this is bad OO design, however it exposes the returned results in an obvious way that is easy to read without chasing javadoc inheritance chains.